sirepo/pkcli/job_agent.py

```diff
@@ -54,6 +54,7 @@ def start():
 
     cfg = pkconfig.init(
         agent_id=pkconfig.Required(str, 'id of this agent'),
+        fastcgi_sock_dir=('/var/tmp', str, 'directory where fastcgi socket will be placed'),
```

Comment:
`pkio.py_path('/var/tmp'), pkio.py_path,`

I think we might actually want to use `/tmp`. We don't want the sockets lying around on reboots of virtual machines. Also, it's shorter, which matters since agent_id is long.

On NERSC, we have a problem because the agent is running natively and job_cmd is in shifter. Please login to Cori and see if /tmp is visible inside and outside the container.

Reply:
https://github.com/radiasoft/sirepo/pull/2773/commits/56c0225deec58e83d9eb23d034a73a052299b95a

I confirmed /tmp is visible inside and outside shifter.